media-libs/opencollada/opencollada-1.6.23.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+EGIT_REPO_URI="https://github.com/KhronosGroup/OpenCOLLADA.git"
+inherit multilib cmake-utils
```

Comment:
It looks as if you inherit multilib for $(get_libdir). In EAPI 6, it's a standard function. So unless you need it for another reason, you don't need multilib.